pkg/cluster/internal/providers/crafting/crafting.go

```diff
@@ -0,0 +1,47 @@
+package crafting
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"sigs.k8s.io/kind/pkg/cluster/internal/providers"
+	"sigs.k8s.io/kind/pkg/cluster/nodeutils"
+	"sigs.k8s.io/kind/pkg/errors"
+)
+
+func InSandboxWorkspace() bool {
+	// TODO. Better way to check.
+	_, ok := os.LookupEnv("SANDBOX_NAME")
```

Comment:
Please check the existence of unix domain socket `/run/sandbox/svc/workspace.sock`